### PR TITLE
Update all collapse triggers with .collapsed class and aria-expanded

### LINKED ISSUE #15741: collapse.js: For multiple triggers targeting the same element, collapsed class/aria-expanded attribute is not applied to all triggers
http://jsbin.com/guyuhemico/1/

### PR DESCRIPTION
**Pull request by cvrebert**
Fixes #15741.
Closes #15751 by merging it.
CC: @hnrch02 for final re-review

### CODE REVIEW

File: js/tests/unit/collapse.js [hunk 1]
```diff
@@ -95,7 +95,41 @@ $(function () {
...
+      .on('shown.bs.collapse', function () {
+        assert.ok(!$target.hasClass('collapsed'), 'target trigger does not have collapsed class')
+        assert.ok(!$alt.hasClass('collapsed'), 'alt trigger does not have collapsed class')
+        done()
+      })
+
+    $target.click()
+  })
+
+  QUnit.test('should add "collapsed" class to all triggers targeting href of target when collapse is hidden', function (assert) {
```
Comment (hnrch02) [inline js/tests/unit/collapse.js]: "targeting href of target" sounds a bit wonky.

**Comment by hnrch02** [comment]
LGTM otherwise.

File: js/tests/unit/collapse.js (hunk 1 above)
Comment (cvrebert) [inline js/tests/unit/collapse.js]: How about just "targeting the collapse"?

Comment (hnrch02) [inline js/tests/unit/collapse.js]: Yeah, that's much clearer.

**Pull request merged**